packages/stream/get-streams/get-streams.test.js

```diff
@@ -0,0 +1,135 @@
+import nock from 'nock'
+import { expect } from 'chai'
+import { getStreams } from './get-streams.js'
+import { Internal } from '../../internal/index.js'
+import { init } from '../../app/init/init.js'
+
+describe('Get Streams Module', function () {
+  describe('Basic test', function () {
+    it('should be a function', function () {
+      expect(getStreams).to.be.a('function')
+    })
+  })
+
+  describe('Negative test', function () {
+    it('should return error if initialization instance not input / not match format', async function () {
+      try {
+        await getStreams()
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('TypeError')
+        expect(error.message).to.be.equal(
+          'Failed to process because initialization is not valid. Please provide required initialization argument which is the initialization instance returned by the init() function'
+        )
+      }
+
+      try {
+        await getStreams({})
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('TypeError')
+        expect(error.message).to.be.equal(
+          'Failed to process because initialization is not valid. Please provide required initialization argument which is the initialization instance returned by the init() function'
+        )
+      }
+
+      try {
+        await getStreams('blabla')
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('TypeError')
+        expect(error.message).to.be.equal(
+          'Failed to process because initialization is not valid. Please provide required initialization argument which is the initialization instance returned by the init() function'
+        )
+      }
+
+      try {
+        await getStreams({ tes: 'blabla' })
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('TypeError')
+        expect(error.message).to.be.equal(
+          'Failed to process because initialization is not valid. Please provide required initialization argument which is the initialization instance returned by the init() function'
+        )
+      }
+    })
+
+    it('should return error response if API key is not valid', async function () {
+      nock(`${Internal.config.api.base_url}`)
+        .get(`/${Internal.config.api.version}/streams/`)
+        .reply(403, { code: 403, message: 'API key is not valid', data: '' })
+
+      try {
+        await getStreams(init({ api_key: 'blabla' }))
+      } catch (error) {
```

Comment:
Same with the [comment on this PR](https://github.com/inlivedev/inlive-js-sdk/pull/12/files#r973824303). We shouldn't use `try catch` for testing